utensor_cgen/transformer/quantize.py

```diff
@@ -1,37 +0,0 @@
-"""Legacy, DON'T USE
```

Comment:
May I please confirm the intent behind deprecating the quantization transformation?  Is it intended that the user will provide a pre-quantized model?  Other than tflite int8 quantization, which external quantization mechanisms are recommended?

Reply:
Yes, it's intended that user can have option to pass quantized or pre-quantized model.
And yes, I simply forget to remove this one in this PR.

We are not deprecate quantization but this transformer is almost of no use in TF 2.x
We have some breaking changes, due to TF 2.x, to the `uTensorGraph` so the conversion to `tf.GraphDef` is broken.
Such conversion is required for this transformer and the `TransformGraph` in TF is removed in TF 2.

Reply:
Ok. Thanks for confirming